Project/pom.xml

```diff
@@ -35,6 +35,13 @@
             <artifactId>jquery</artifactId>
             <version>1.9.1</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.webjars/font-awesome -->
+        <dependency>
+            <groupId>org.webjars</groupId>
+            <artifactId>font-awesome</artifactId>
+            <version>4.7.0</version>
+        </dependency>
+
         <dependency>
```

Comment:
https://mvnrepository.com/artifact/org.webjars/font-awesome/5.7.2, dont use old version, you an upto date one please

Reply:
Latest version didn't work. Literally the logo just didn't show up. This one was the latest I could use.

Reply:
If we wanted to use latest versions as well, then we have outdated versions of jquery and bootstrap imported in here.

Reply:
Ya we should grab the latest, font

https://fontawesome.com/icons/trash-alt?style=solid

youre using fa as your class prefix, thats old

far/fas are new 

<i class="fas fa-trash-alt"></i>